### ISSUE TITLE
Support CoreOS

### ISSUE THREAD

**Issue by parnurzeal** [cobbler/cobbler#1338]
Hi,

Just wondering if I can use Cobbler to install CoreOS ?

Thanks,

**Comment by FlorianHeigl** [comment]
@stresler are you still poking at this?

**Comment by JeremyMordkoff** [comment]
This seems like you should be pretty straight forward but afaict nobody's documented  the steps

**Comment by SchoolGuy** [comment]
@JeremyMordkoff Yes it is indeed but we don't have much contributors and thus we do the most important things first. However I will try to do something about this this weekend.

**Comment by JeremyMordkoff** [comment]
Thank you! If there is something I can do, please let me know. I've been using cobbler for a long time (I'm almost afraid to tell you what version I'm running), and it's about time I helped out. I know all of the protocols, linux, python, bash, perl... At a minimum, I will do some testing and provide feedback on any documentation.

**Comment by SchoolGuy** [comment]
@JeremyMordkoff Every help you are interested to give is appreciated. Be it feedback for the docs, contributions to the website, contributions to the core or whatever you can give.

Support for iginition files support is not explicitly there but as always you can create templates and in the end Cobbler should not care as long as it is setup correctly.

**Comment by SchoolGuy** [comment]
@JeremyMordkoff Maybe you were wondering why nothing happend explicitly. I am creating (according to our roadmap) a library (https://github.com/cobbler/libcobblersignatures) which enables us to not do this by hand but instead manipulate and update the file automatically to ease the pain of maintaining this file and improve the self maintenance in case you have your own file.

**Comment by FlorianHeigl** [comment]
@SchoolGuy since I asked that long ago - the README there looks good. I would probably be able to guess what to do after some time. Thank you for that!

if you can, please make them more explicit once done: "this is what we expect people to do that want to contribute a distro sig upstream, this is how they can validate if they did it well, this is a basic rule for random weird operating systems", so finally we get to the point where 'we' can contribute to this bit that is the most scalable :-)

**Comment by SchoolGuy** [comment]
@FlorianHeigl Well that is the plan in the very end. First of all I need to get the lib done and then I can write that. The ultimate test if it has worked will be ultimately if the distro can be imported. The problematic thing is that netinstaller are still an unsolved generic problem in Cobbler but that is another issue.
